build-linux.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Build LK8000 for Linux only, then copy executable to Common/Distribution/LK8000.
+# Usage: ./build-linux.sh
+set -e
+SCRIPTDIR="$(cd "$(dirname "$0")" && pwd)"
+cd "$SCRIPTDIR"
+make -j$(nproc) TARGET=LINUX "$@"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)